### ISSUE TITLE
Crash on exiting

### ISSUE THREAD

**Issue by mxmilkiib** [falkTX/Carla#1030]
```
           PID: 28580 (carla)
           UID: 1000 (milk)
           GID: 1000 (milk)
        Signal: 6 (ABRT)
     Timestamp: Fri 2020-02-21 14:19:26 GMT (22s ago)
  Command Line: /usr/bin/python3 /usr/share/carla/carla --with-appname=/usr/bin/carla --with-libprefix=/usr drone_winds_on.carxp
    Executable: /usr/bin/python3.8
 Control Group: /user.slice/user-1000.slice/session-1.scope
          Unit: session-1.scope
         Slice: user-1000.slice
       Session: 1
     Owner UID: 1000 (milk)
       Boot ID: 0f47849a445c42f282f30753bbe301bb
    Machine ID: 868484078e134905a6dcbc0fdd2079bb
      Hostname: rose.local
       Storage: /var/lib/systemd/coredump/core.carla.1000.0f47849a445c42f282f30753bbe301bb.28580.1582294766000000000000.lz4
       Message: Process 28580 (carla) of user 1000 dumped core.
                
                Stack trace of thread 28580:
                #0  0x00007f80db6f5ce5 raise (libc.so.6 + 0x3bce5)
                #1  0x00007f80db6df857 abort (libc.so.6 + 0x25857)
                #2  0x00007f80db7392b0 __libc_message (libc.so.6 + 0x7f2b0)
                #3  0x00007f80db74074a malloc_printerr (libc.so.6 + 0x8674a)
                #4  0x00007f80db7409fc munmap_chunk (libc.so.6 + 0x869fc)
                #5  0x00007f80c6e867c4 n/a (libcarla_standalone2.so + 0x1537c4)
                #6  0x00007f80c6dd6ab5 _ZN12CarlaBackend11CarlaPlugin13ProtectedData12PostRtEventsD1Ev (libcarla_standalone2.so + 0xa3ab5)
                #7  0x00007f80c6dd7c9c _ZN12CarlaBackend11CarlaPlugin13ProtectedDataD1Ev (libcarla_standalone2.so + 0xa4c9c)
                #8  0x00007f80c6dcc7a1 _ZN12CarlaBackend11CarlaPluginD2Ev (libcarla_standalone2.so + 0x997a1)
                #9  0x00007f80c6e2a429 n/a (libcarla_standalone2.so + 0xf7429)
                #10 0x00007f80c6d87dcc _ZN12CarlaBackend11CarlaEngine16removeAllPluginsEv (libcarla_standalone2.so + 0x54dcc)
                #11 0x00007f80d3fc369a ffi_call_unix64 (libffi.so.6 + 0x669a)
                #12 0x00007f80d3fc2fb6 ffi_call (libffi.so.6 + 0x5fb6)
                #13 0x00007f80ce8df11d n/a (_ctypes.cpython-38-x86_64-linux-gnu.so + 0xf11d)
                #14 0x00007f80ce8e3767 n/a (_ctypes.cpython-38-x86_64-linux-gnu.so + 0x13767)
                #15 0x00007f80db431ad2 _PyObject_MakeTpCall (libpython3.8.so.1.0 + 0x136ad2)
                #16 0x00007f80db4ee7f4 _PyEval_EvalFrameDefault (libpython3.8.so.1.0 + 0x1f37f4)
                #17 0x00007f80db4d87a6 n/a (libpython3.8.so.1.0 + 0x1dd7a6)
                #18 0x00007f80db4ee3c8 _PyEval_EvalFrameDefault (libpython3.8.so.1.0 + 0x1f33c8)
                #19 0x00007f80db4d87a6 n/a (libpython3.8.so.1.0 + 0x1dd7a6)
                #20 0x00007f80db4ee3c8 _PyEval_EvalFrameDefault (libpython3.8.so.1.0 + 0x1f33c8)
                #21 0x00007f80db4d87a6 n/a (libpython3.8.so.1.0 + 0x1dd7a6)
                #22 0x00007f80db4ee3c8 _PyEval_EvalFrameDefault (libpython3.8.so.1.0 + 0x1f33c8)
                #23 0x00007f80db4d6e3b _PyEval_EvalCodeWithName (libpython3.8.so.1.0 + 0x1dbe3b)
                #24 0x00007f80db4d8892 n/a (libpython3.8.so.1.0 + 0x1dd892)
                #25 0x00007f80db4ee3c8 _PyEval_EvalFrameDefault (libpython3.8.so.1.0 + 0x1f33c8)
                #26 0x00007f80db4d806d _PyFunction_Vectorcall (libpython3.8.so.1.0 + 0x1dd06d)
                #27 0x00007f80db4d8a7b n/a (libpython3.8.so.1.0 + 0x1dda7b)
                #28 0x00007f80db42930d PyObject_Call (libpython3.8.so.1.0 + 0x12e30d)
                #29 0x00007f80dab777b1 n/a (QtCore.abi3.so + 0x2117b1)
                #30 0x00007f80dab77c81 n/a (QtCore.abi3.so + 0x211c81)
                #31 0x00007f80dab77e2d n/a (QtCore.abi3.so + 0x211e2d)
                #32 0x00007f80dab7a7ae n/a (QtCore.abi3.so + 0x2147ae)
                #33 0x00007f80dab7a892 n/a (QtCore.abi3.so + 0x214892)
                #34 0x00007f80da3ba1b2 n/a (libQt5Core.so.5 + 0x2db1b2)
                #35 0x00007f80dab742c0 n/a (QtCore.abi3.so + 0x20e2c0)
                #36 0x00007f80db469b07 n/a (libpython3.8.so.1.0 + 0x16eb07)
                #37 0x00007f80db4ea0ce _PyEval_EvalFrameDefault (libpython3.8.so.1.0 + 0x1ef0ce)
                #38 0x00007f80db4d806d _PyFunction_Vectorcall (libpython3.8.so.1.0 + 0x1dd06d)
                #39 0x00007f80db42930d PyObject_Call (libpython3.8.so.1.0 + 0x12e30d)
                #40 0x00007f80db536f16 n/a (libpython3.8.so.1.0 + 0x23bf16)
                #41 0x00007f80db536fb5 n/a (libpython3.8.so.1.0 + 0x23bfb5)
                #42 0x00007f80db407310 n/a (libpython3.8.so.1.0 + 0x10c310)
                #43 0x00007f80db4d806d _PyFunction_Vectorcall (libpython3.8.so.1.0 + 0x1dd06d)
                #44 0x00007f80db4d8a7b n/a (libpython3.8.so.1.0 + 0x1dda7b)
                #45 0x00007f80db42930d PyObject_Call (libpython3.8.so.1.0 + 0x12e30d)
                #46 0x00007f80dab777b1 n/a (QtCore.abi3.so + 0x2117b1)
                #47 0x00007f80dab77c81 n/a (QtCore.abi3.so + 0x211c81)
                #48 0x00007f80dab77e2d n/a (QtCore.abi3.so + 0x211e2d)
                #49 0x00007f80dab7a7ae n/a (QtCore.abi3.so + 0x2147ae)
                #50 0x00007f80dab7a57d n/a (QtCore.abi3.so + 0x21457d)
                #51 0x00007f80dab7a892 n/a (QtCore.abi3.so + 0x214892)
                #52 0x00007f80da3b226a _ZN7QObject5eventEP6QEvent (libQt5Core.so.5 + 0x2d326a)
                #53 0x00007f80d5a23489 _ZN7QWidget5eventEP6QEvent (libQt5Widgets.so.5 + 0x19f489)
                #54 0x00007f80d5ad4f3f _ZN6QFrame5eventEP6QEvent (libQt5Widgets.so.5 + 0x250f3f)
                #55 0x00007f80d63047e4 n/a (QtWidgets.abi3.so + 0x3847e4)
                #56 0x00007f80d59df362 _ZN19QApplicationPrivate13notify_helperEP7QObjectP6QEvent (libQt5Widgets.so.5 + 0x15b362)
                #57 0x00007f80d59e8819 _ZN12QApplication6notifyEP7QObjectP6QEvent (libQt5Widgets.so.5 + 0x164819)
                #58 0x00007f80d62f272f n/a (QtWidgets.abi3.so + 0x37272f)
                #59 0x00007f80da384962 _ZN16QCoreApplication15notifyInternal2EP7QObjectP6QEvent (libQt5Core.so.5 + 0x2a5962)
                #60 0x00007f80da3871c6 _ZN23QCoreApplicationPrivate16sendPostedEventsEP7QObjectiP11QThreadData (libQt5Core.so.5 + 0x2a81c6)
                #61 0x00007f80da3dd9d4 n/a (libQt5Core.so.5 + 0x2fe9d4)
                #62 0x00007f80d997939e g_main_context_dispatch (libglib-2.0.so.0 + 0x6a39e)
                #63 0x00007f80d997b1b1 n/a (libglib-2.0.so.0 + 0x6c1b1)
                
                Stack trace of thread 28726:
                #0  0x00007f80db7b3f8d syscall (libc.so.6 + 0xf9f8d)
                #1  0x00007f80c6b189fb n/a (libjack.so.0 + 0x2e9fb)
                #2  0x00007f80c6aff155 n/a (libjack.so.0 + 0x15155)
                #3  0x00007f80c6afe217 n/a (libjack.so.0 + 0x14217)
                #4  0x00007f80c6afda28 n/a (libjack.so.0 + 0x13a28)
                #5  0x00007f80c6b16bdd n/a (libjack.so.0 + 0x2cbdd)
                #6  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #7  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28609:
                #0  0x00007f80db7b3f8d syscall (libc.so.6 + 0xf9f8d)
                #1  0x00007f80c6b189fb n/a (libjack.so.0 + 0x2e9fb)
                #2  0x00007f80c6aff155 n/a (libjack.so.0 + 0x15155)
                #3  0x00007f80c6afe217 n/a (libjack.so.0 + 0x14217)
                #4  0x00007f80c6afda28 n/a (libjack.so.0 + 0x13a28)
                #5  0x00007f80c6b16bdd n/a (libjack.so.0 + 0x2cbdd)
                #6  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #7  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28591:
                #0  0x00007f80db7aeabf __poll (libc.so.6 + 0xf4abf)
                #1  0x00007f80d997b120 n/a (libglib-2.0.so.0 + 0x6c120)
                #2  0x00007f80d997b1f1 g_main_context_iteration (libglib-2.0.so.0 + 0x6c1f1)
                #3  0x00007f80da3dd020 _ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE (libQt5Core.so.5 + 0x2fe020)
                #4  0x00007f80da3834cc _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE (libQt5Core.so.5 + 0x2a44cc)
                #5  0x00007f80da1a7e52 _ZN7QThread4execEv (libQt5Core.so.5 + 0xc8e52)
                #6  0x00007f80d508db28 n/a (libQt5DBus.so.5 + 0x15b28)
                #7  0x00007f80da1a8fc6 n/a (libQt5Core.so.5 + 0xc9fc6)
                #8  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #9  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28737:
                #0  0x00007f80db7b3f8d syscall (libc.so.6 + 0xf9f8d)
                #1  0x00007f80c6b189fb n/a (libjack.so.0 + 0x2e9fb)
                #2  0x00007f80c6aff155 n/a (libjack.so.0 + 0x15155)
                #3  0x00007f80c6afe217 n/a (libjack.so.0 + 0x14217)
                #4  0x00007f80c6afda28 n/a (libjack.so.0 + 0x13a28)
                #5  0x00007f80c6b16bdd n/a (libjack.so.0 + 0x2cbdd)
                #6  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #7  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28592:
                #0  0x00007f80db2e8cf5 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xfcf5)
                #1  0x00007f80da1aecb4 _ZN14QWaitCondition4waitEP6QMutex14QDeadlineTimer (libQt5Core.so.5 + 0xcfcb4)
                #2  0x00007f80da1aed92 _ZN14QWaitCondition4waitEP6QMutexm (libQt5Core.so.5 + 0xcfd92)
                #3  0x00007f80d5c1b6c6 n/a (libQt5Widgets.so.5 + 0x3976c6)
                #4  0x00007f80da1a8fc6 n/a (libQt5Core.so.5 + 0xc9fc6)
                #5  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #6  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28594:
                #0  0x00007f80db2e8cf5 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xfcf5)
                #1  0x00007f80bf800e1c n/a (i965_dri.so + 0x4f8e1c)
                #2  0x00007f80bf800a18 n/a (i965_dri.so + 0x4f8a18)
                #3  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #4  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28648:
                #0  0x00007f80db7aeabf __poll (libc.so.6 + 0xf4abf)
                #1  0x00007f80d997b120 n/a (libglib-2.0.so.0 + 0x6c120)
                #2  0x00007f80d997b1f1 g_main_context_iteration (libglib-2.0.so.0 + 0x6c1f1)
                #3  0x00007f80d997b242 n/a (libglib-2.0.so.0 + 0x6c242)
                #4  0x00007f80d9957bb1 n/a (libglib-2.0.so.0 + 0x48bb1)
                #5  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #6  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28728:
                #0  0x00007f80db7b3f8d syscall (libc.so.6 + 0xf9f8d)
                #1  0x00007f80c6b189fb n/a (libjack.so.0 + 0x2e9fb)
                #2  0x00007f80c6aff155 n/a (libjack.so.0 + 0x15155)
                #3  0x00007f80c6afe217 n/a (libjack.so.0 + 0x14217)
                #4  0x00007f80c6afda28 n/a (libjack.so.0 + 0x13a28)
                #5  0x00007f80c6b16bdd n/a (libjack.so.0 + 0x2cbdd)
                #6  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #7  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28590:
                #0  0x00007f80db7aeabf __poll (libc.so.6 + 0xf4abf)
                #1  0x00007f80d6510630 n/a (libxcb.so.1 + 0xc630)
                #2  0x00007f80d65122db xcb_wait_for_event (libxcb.so.1 + 0xe2db)
                #3  0x00007f80cd8edb81 n/a (libQt5XcbQpa.so.5 + 0x65b81)
                #4  0x00007f80da1a8fc6 n/a (libQt5Core.so.5 + 0xc9fc6)
                #5  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #6  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28735:
                #0  0x00007f80db7b3f8d syscall (libc.so.6 + 0xf9f8d)
                #1  0x00007f80c6b189fb n/a (libjack.so.0 + 0x2e9fb)
                #2  0x00007f80c6aff155 n/a (libjack.so.0 + 0x15155)
                #3  0x00007f80c6afe217 n/a (libjack.so.0 + 0x14217)
                #4  0x00007f80c6afda28 n/a (libjack.so.0 + 0x13a28)
                #5  0x00007f80c6b16bdd n/a (libjack.so.0 + 0x2cbdd)
                #6  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #7  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28736:
                #0  0x00007f80db7aa4fc __read (libc.so.6 + 0xf04fc)
                #1  0x00007f80c6b17e2f n/a (libjack.so.0 + 0x2de2f)
                #2  0x00007f80c6b1cd19 n/a (libjack.so.0 + 0x32d19)
                #3  0x00007f80c6b16bdd n/a (libjack.so.0 + 0x2cbdd)
                #4  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #5  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28593:
                #0  0x00007f80db2e8cf5 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xfcf5)
                #1  0x00007f80bf800e1c n/a (i965_dri.so + 0x4f8e1c)
                #2  0x00007f80bf800a18 n/a (i965_dri.so + 0x4f8a18)
                #3  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #4  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28595:
                #0  0x00007f80db2e8cf5 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xfcf5)
                #1  0x00007f80bf800e1c n/a (i965_dri.so + 0x4f8e1c)
                #2  0x00007f80bf800a18 n/a (i965_dri.so + 0x4f8a18)
                #3  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #4  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28604:
                #0  0x00007f80db2e901a pthread_cond_timedwait@@GLIBC_2.3.2 (libpthread.so.0 + 0x1001a)
                #1  0x00007f80c6eba497 _ZNSt6thread11_State_implINS_8_InvokerISt5tupleIJZN7ableton9platforms26LockFreeCallbackDispatcherISt8functionIFvvEENSt6chrono8durationIlSt5ratioILl1ELl1000EEEEEC4ES8_SD_EUlvE_EEEEE6_M_runEv (libcar>
                #2  0x00007f80da7f0bc4 execute_native_thread_routine (libstdc++.so.6 + 0xcfbc4)
                #3  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #4  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28596:
                #0  0x00007f80db2e8cf5 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xfcf5)
                #1  0x00007f80bf800e1c n/a (i965_dri.so + 0x4f8e1c)
                #2  0x00007f80bf800a18 n/a (i965_dri.so + 0x4f8a18)
                #3  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #4  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28725:
                #0  0x00007f80db7aa4fc __read (libc.so.6 + 0xf04fc)
                #1  0x00007f80c6b17e2f n/a (libjack.so.0 + 0x2de2f)
                #2  0x00007f80c6b1cd19 n/a (libjack.so.0 + 0x32d19)
                #3  0x00007f80c6b16bdd n/a (libjack.so.0 + 0x2cbdd)
                #4  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #5  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28605:
                #0  0x00007f80db7b970e epoll_wait (libc.so.6 + 0xff70e)
                #1  0x00007f80c6ebc71e n/a (libcarla_standalone2.so + 0x18971e)
                #2  0x00007f80c6ebd406 n/a (libcarla_standalone2.so + 0x18a406)
                #3  0x00007f80c6ec7f6d _ZNSt6thread11_State_implINS_8_InvokerISt5tupleIJZN7ableton9platforms4asio7ContextINS4_5posix13ScanIpIfAddrsENS3_4util7NullLogEEC4INS3_4link10ControllerISt8functionIFvmEESF_IFvNSD_5TempoEEESF_IFvbE>
                #4  0x00007f80da7f0bc4 execute_native_thread_routine (libstdc++.so.6 + 0xcfbc4)
                #5  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #6  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28606:
                #0  0x00007f80db2e8cf5 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0 + 0xfcf5)
                #1  0x00007f80c6b176b0 n/a (libjack.so.0 + 0x2d6b0)
                #2  0x00007f80c6b0eed6 n/a (libjack.so.0 + 0x24ed6)
                #3  0x00007f80c6b16bdd n/a (libjack.so.0 + 0x2cbdd)
                #4  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #5  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28603:
                #0  0x00007f80db7b970e epoll_wait (libc.so.6 + 0xff70e)
                #1  0x00007f80c6ebc71e n/a (libcarla_standalone2.so + 0x18971e)
                #2  0x00007f80c6ebd406 n/a (libcarla_standalone2.so + 0x18a406)
                #3  0x00007f80c6ec12b1 _ZNSt6thread11_State_implINS_8_InvokerISt5tupleIJZN7ableton9platforms4asio7ContextINS4_5posix13ScanIpIfAddrsENS3_4util7NullLogEEC4INSB_14DefaultHandlerEEET_EUlRN4asio10io_contextESD_E_St17reference>
                #4  0x00007f80da7f0bc4 execute_native_thread_routine (libstdc++.so.6 + 0xcfbc4)
                #5  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #6  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28664:
                #0  0x00007f80db7aeabf __poll (libc.so.6 + 0xf4abf)
                #1  0x00007f80d4690c14 n/a (libpulse.so.0 + 0xdc14)
                #2  0x00007f80d469e059 pa_mainloop_poll (libpulse.so.0 + 0x1b059)
                #3  0x00007f80d46a8301 pa_mainloop_iterate (libpulse.so.0 + 0x25301)
                #4  0x00007f80d46a83b1 pa_mainloop_run (libpulse.so.0 + 0x253b1)
                #5  0x00007f80d469861e n/a (libpulse.so.0 + 0x1561e)
                #6  0x00007f80d461cd1c n/a (libpulsecommon-13.0.so + 0x1dd1c)
                #7  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #8  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28727:
                #0  0x00007f80db7aa4fc __read (libc.so.6 + 0xf04fc)
                #1  0x00007f80c6b17e2f n/a (libjack.so.0 + 0x2de2f)
                #2  0x00007f80c6b1cd19 n/a (libjack.so.0 + 0x32d19)
                #3  0x00007f80c6b16bdd n/a (libjack.so.0 + 0x2cbdd)
                #4  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #5  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28610:
                #0  0x00007f80db7812d1 clock_nanosleep@@GLIBC_2.17 (libc.so.6 + 0xc72d1)
                #1  0x00007f80db786bf7 __nanosleep (libc.so.6 + 0xccbf7)
                #2  0x00007f80db7b1799 usleep (libc.so.6 + 0xf7799)
                #3  0x00007f80c6dbcda2 n/a (libcarla_standalone2.so + 0x89da2)
                #4  0x00007f80c6d6d05f n/a (libcarla_standalone2.so + 0x3a05f)
                #5  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #6  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28607:
                #0  0x00007f80db7aa4fc __read (libc.so.6 + 0xf04fc)
                #1  0x00007f80c6b17e2f n/a (libjack.so.0 + 0x2de2f)
                #2  0x00007f80c6b1cd19 n/a (libjack.so.0 + 0x32d19)
                #3  0x00007f80c6b16bdd n/a (libjack.so.0 + 0x2cbdd)
                #4  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #5  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28611:
                #0  0x00007f80db7812d1 clock_nanosleep@@GLIBC_2.17 (libc.so.6 + 0xc72d1)
                #1  0x00007f80db786bf7 __nanosleep (libc.so.6 + 0xccbf7)
                #2  0x00007f80db7b1799 usleep (libc.so.6 + 0xf7799)
                #3  0x00007f80c6d6dfeb n/a (libcarla_standalone2.so + 0x3afeb)
                #4  0x00007f80c6d6d05f n/a (libcarla_standalone2.so + 0x3a05f)
                #5  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #6  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
                
                Stack trace of thread 28734:
                #0  0x00007f80db7aa4fc __read (libc.so.6 + 0xf04fc)
                #1  0x00007f80c6b17e2f n/a (libjack.so.0 + 0x2de2f)
                #2  0x00007f80c6b1cd19 n/a (libjack.so.0 + 0x32d19)
                #3  0x00007f80c6b16bdd n/a (libjack.so.0 + 0x2cbdd)
                #4  0x00007f80db2e246f start_thread (libpthread.so.0 + 0x946f)
                #5  0x00007f80db7b93d3 __clone (libc.so.6 + 0xff3d3)
```

**Comment by mxmilkb** [comment]
[Here](https://gist.github.com/mxmilkb/d2726a53ea1934463d8241503f68a1c8)'s a slightly different segfault on exit.

**Comment by falkTX** [comment]
how do you reproduce this?

**Comment by falkTX** [comment]
Pretty sure this was fixed in 2.3.1, if that is not the case please reopen.
